suites/upgrade/infernalis-x/stress-split/5-workload/snaps-few-objects.yaml

```diff
@@ -0,0 +1,18 @@
+meta:
+- desc: |
+   add test details here
```

Comment:
randomized correctness test for rados operations on a replicated pool with snapshot operations